README.md

```diff
@@ -1 +1,77 @@
-# crypto-price-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
Please add `project name` and give appropriate credit to the design author in the `Acknowledgements` section of README.